Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+                                    // The walkthroughSelector is later defined and orchardCoreAdminWalkthrough is only used later.
+                                    // eslint-disable-next-line no-use-before-define
+                                    walkthroughSelector.start();
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'welcome',
+                    },
+                    {
+                        title: 'Setup recipe',
+                        text: 'The setup recipe in <a href="https://github.com/Lombiq/Orchard-Walkthroughs">Lombiq.Walkthroughs module</a>' +
+                            '(named Walkthroughs) used the <a href="https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json">Blog recipe</a>' +
+                            ' as a base recipe. In Orchard Core, a <a href="https://docs.orchardcore.net/en/main/docs/reference/modules/Recipes/">recipe</a> ' +
+                            'is a json file that defines a set of instructions for setting up and configuring an' +
+                            ' Orchard Core application. Recipes can include predefined content types, widgets, menus, content items, and other' +
+                            'configurations.They are used to streamline the setup of an Orchard Core site, making it easier ' +
+                            '  to create consistent site structures and content. Recipes can be executed during the initial' +
+                            ' setup of a site or at any point to apply configurations or import content.',
+                        buttons: [
+                            backButton,
+                            nextButton,
+                        ],
+                        id: 'setup_recipe',
+                    },
+                    {
+                        title: 'Site setup',
+                        text: 'To get to this same point (a set up site), first you will need to get through the' +
+                            ' <a href="https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json">' +
+                            'setup screen</a>. You can also select there the before mentioned setup recipe.',
+                        buttons: [
+                            backButton,
+                            nextButton,
+                        ],
+                        id: 'site_setup',
+                    },
+                    {
+                        title: 'Log in',
+                        text: 'Let\'s log in! Please go to the following URL <i>"~/Login"</i>, by clicking on the' +
+                            ' <i>"Next"</i> button.',
```

Comment:
Clicking on this next button redirects to this link while doing it on a "walkthroughs" tenant, and ofc giving 404:
https://localhost:44301/walkthroughsLogin?shepherdTour=orchardCoreAdminWalkthrough&shepherdStep=login_page

Reply:
I tried it, but it works for me. Could you help me with repro?

![giff](https://github.com/Lombiq/Orchard-Walkthroughs/assets/80963259/67083500-56d9-486b-8025-a1b32a7d4b1b)

Reply:
Looks okay now.

Now I've met with the following problem. When I try to select a Tag in the blog editor the selector disappears after a short time and you can't select anything but the bottom option, but only if you are fast enough:
https://www.loom.com/share/e2f3ec291b724a6588c4a3cc702a3a39?sid=5db00369-b492-4598-84d7-673e9c147960

Reply:
This seems like the same bug as here: https://github.com/Lombiq/Orchard-Walkthroughs/pull/2#discussion_r1448695381

I tried workarounds, but it seems like sheperd.js focus ruins this search box for some reason. Is inputting the tag here too a good solution?

Reply:
Never mind, I found a property on the dialog window, that was causing this.

Reply:
It does no disappear now, but you can only select "Space":
![image](https://github.com/Lombiq/Orchard-Walkthroughs/assets/19938236/e4d56059-8fab-44ec-a875-f0021f2bac6c)